src/modules/template/service.test.js

```diff
@@ -95,4 +95,34 @@ describe('test suite for TemplateService', () => {
       expect(templates[0]).toMatchObject(mockTemplate1)
     })
   })
+
+  describe('test suite for TemplateService.updateTarget()', () => {
+    test('update target', async () => {
```

Comment:
Êu nhớ sửa hết những chỗ còn kiểu như này nhé. Rõ ràng route này có update những cái khác đâu

```suggestion
    test('update target CSS selector', async () => {
```